packages/multi-select-combo-box/test/basic.test.js

```diff
@@ -489,6 +460,114 @@ describe('basic', () => {
         expect(chips[2].hasAttribute('readonly')).to.be.true;
       });
     });
+
+    describe('keyboard navigation', () => {
+      beforeEach(async () => {
+        comboBox.selectedItems = ['apple', 'banana'];
+        comboBox.style.width = '300px';
+        await nextResize(comboBox);
+        inputElement.focus();
+      });
+
+      describe('Backspace', () => {
+        it('should not remove last chip on Backspace but mark it as focused', async () => {
+          await sendKeys({ down: 'Backspace' });
+          const chips = getChips(comboBox);
+          expect(chips.length).to.equal(3);
+          expect(chips[1].hasAttribute('focused')).to.be.false;
+          expect(chips[2].hasAttribute('focused')).to.be.true;
+        });
+
+        it('should remove last chip on subsequent Backspace after focusing', async () => {
+          await sendKeys({ down: 'Backspace' });
+          await sendKeys({ down: 'Backspace' });
```

Comment:
To avoid potential side-effects on the following tests, should we consider `press` instead of `down`?

```suggestion
          await sendKeys({ press: 'Backspace' });
```

Reply:
Done.